RELEASE_NOTES.md

```diff
@@ -180,7 +184,19 @@ bin/rails decidim:upgrade:fix_nickname_casing
 
 You can read more about this change on PR [#14272](https://github.com/decidim/decidim/pull/14272).
 
-### 3.4. [[TITLE OF THE ACTION]]
+### 3.4. Removal of secrets system
+
+If you were already using the Environment Variables for the configuration of your application, then you can remove both the config/secrets.yml and also the decidim initializer:
+If you are not using the ENV system, you will need to adjust your application settings to use it.
+
+Before actually removing the initializer, just make sure you do not have any custom configuration.
+
+```bash
+git rm config/secrets.yml
+git rm config/initializers/decidim.rb
```

Comment:
We should add a command to replacing the config/storage.yml with wget (from [decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb](https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb))

```suggestion
git rm config/initializers/decidim.rb
wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb -O config/storage.yml
```